src/containers/unit_detail.js

```diff
@@ -0,0 +1,112 @@
+import React, {Component} from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+/**
+ * container UnitDetail:
+ * represents the detial of selected unit
+ */
+class UnitDetail extends Component {
+  render () {
+    const units = this.props.allUnits
+    const results = this.props.searchResultUnits
+    const unit = this.props.activeUnit
+
+    /**
+     * return empty div when:
+     * all units haven't been fetched or
+     * the search hasn't been kicked off or
+     * there's no units in the search result
+    */
+    if (!units || !results || results.length === 0) {
```

Comment:
You don't need to document everything. Just document the functions that you use, or functions that you need to reuse